charts/stable/whisper-asr-webservice/values.yaml

```diff
@@ -20,7 +20,7 @@ workload:
           imageSelector: image
           probes:
             liveness:
-              type: http
+              type: tcp
               path: "/docs"
```

Comment:
There is no path for tcp..

Also startup can be removed completely and it will default to tcp (as it should be)